docs/using-radicle/running-a-seed-node.md

```diff
@@ -11,7 +11,7 @@ Conceptually, they are similar to pubs in [Secure Scuttlebutt][ss].
 To build and run a seed node, you'll have to install some prerequisites on your
 machine first:
 
-  - [Rust toolchain][ru]
+  - [Rust toolchain][ru], you'll need **nightly**
```

Comment:
We could drop a line about rustup.

Reply:
Just that you have to install rustup? 

https://rustup.rs/

Reply:
More that you could use rustup to achieve getting hold of the nightly toolchain required.

Reply:
https://github.com/radicle-dev/radicle-docs/pull/94/commits/ff7b7b6a089700d5eb8c288cb38bdbfa781d7519

Reply:
Dope!